src/settings.js

```diff
@@ -1232,10 +1232,17 @@ var IN_TEST_HARNESS = 0;
 // If true, enables support for pthreads.
 var USE_PTHREADS = 0;
 
-// Specifies the number of web workers that are preallocated before runtime is
-// initialized. If 0, workers are created on demand.
+// Specifies the number of web workers that are preallocated with the
+// webassembly module before runtime is initialized. If 0, workers are
+// created on demand.
 var PTHREAD_POOL_SIZE = 0;
 
+// Specifies the number of web workers that are preallocated before the
+// webassembly module is compiled. If 0, workers are created on demand. You
```

Comment:
`compiled` => `compiled on the main thread`.

Reply:
Done.